client/modules/module_ticker.js

```diff
@@ -42,15 +43,15 @@ define(function(require) {
   window.requestAnimationFrame(draw);
 
   return {
-    add: function(module, globals) {
-      modulesToDraw.push({module: module, globals: globals});
+    add: function(name, module, globals) {
+      modulesToDraw.push({name, module, globals});
```

Comment:
Good god, I didn't know {a,b,c} syntax existed.  Where have you been all my life?!?